packaging/python/test_wheels.sh

```diff
@@ -166,7 +166,12 @@ else
   echo " == Using global install == "
 fi
 
-# install neuron and neuron
+# python 3.6 needs updated pip
+if [[ "$python_ver" == "36" ]]; then
+  $python_exe -m pip install --upgrade pip
```

Comment:
yes, old pips dont understand the ML2014 format. I think it's good to keep compat.

Reply:
Do you know if we could inform 3.6 users somehow when they do `pip install neuron` that they must upgrade their pip? I imagine not. Will have to settle for a release note in that case.

Reply:
No, we don't have hooks for when installing wheels, it's basically an unzip and done.

Reply:
I would say python people are now more or less familiar with this problem i.e. pip upgrade. So no show stopper if they have to explicitly do this.